prql-compiler/src/lib.rs

```diff
@@ -7,13 +7,16 @@ mod semantic;
 mod sql;
 mod utils;
 
+use crate::{ast::Item};
```

Comment:
I don't know if you have a convention (eg. should you put imports in alphabetical order, etc.)

Plus, I didn't research the difference between `use` and `pub use`.

Reply:
`cargo fmt` will organize these!